packages/core/src/js/feedback/FeedbackWidget.tsx

```diff
@@ -273,24 +273,27 @@ export class FeedbackWidget extends React.Component<FeedbackWidgetProps, Feedbac
         >
         <View style={styles.container}>
           <View style={styles.titleContainer}>
-            <Text style={styles.title}>{text.formTitle}</Text>
+            <Text style={styles.title} testID='form-title'>{text.formTitle}</Text>
             {config.showBranding && (
               <Image
                 source={{ uri: sentryLogo }}
                 style={styles.sentryLogo}
                 testID='sentry-logo'
+                accessibilityLabel='Sentry logo'
               />
             )}
           </View>
 
           {config.showName && (
           <>
-            <Text style={styles.label}>
+            <Text style={styles.label} testID='name-label' accessibilityLabel={text.nameLabel}>
```

Comment:
Do we need the additional `testId` and `accessibilityLabel`? 

I would prefer not using the `testIDs` if not needed to avoid conflicting with user apps.

If we need them then lets prefix them with something unique like `sentry-feedback-*`.

Reply:
Updated with 8176d4f8424db39684027c68395c05b2211461cd:
- I've cleaned up all the unneeded test-id and used the `sentry-feedback-*` prefix for those used.
- Removed the unneeded accessibility labels.

Reply:
> I would prefer not using the testIDs if not needed to avoid conflicting with user apps.

I ended up using both `testID`'s and text lookups since:

- [On iOS legacy text lookups fail](https://github.com/getsentry/sentry-react-native/actions/runs/14757306115/job/41430627531?pr=4604)
- [On Android RN `0.65.3` input fields lookups fail with `testID`'s](https://github.com/getsentry/sentry-react-native/actions/runs/14754006058/job/41426737189)